locale/zh_CN/libbout.po

```diff
@@ -16,75 +16,679 @@ msgstr ""
+"to the existing output files, otherwise overwrite them\n"
+"  VAR=VALUE\t\tSpecify a VALUE for input parameter VAR\n"
+"\n"
+"For all possible input parameters, see the user manual and/or the physics "
+"model source (e.g. {:s}.cxx)\n"
+msgstr ""
+
+#: ../include/options.hxx:641
+msgid ") overwritten with:"
+msgstr ""
+
+#: ../tests/unit/src/test_bout++.cxx:294
+msgid "4 of 8"
+msgstr ""
+
+#: ../src/sys/options.cxx:463
+msgid "All options used\n"
+msgstr ""
+
+#: ../src/bout++.cxx:408
+msgid "BOUT++ version {:s}\n"
+msgstr ""
+
+#: ../src/bout++.cxx:116
+msgid "Bad command line arguments:\n"
+msgstr ""
+
+#: ../src/mesh/impls/bout/boutmesh.cxx:873
+msgid "Boundary regions in this processor: "
+msgstr ""
+
+#: ../src/mesh/impls/bout/boutmesh.cxx:420
+#: ../src/mesh/impls/bout/boutmesh.cxx:1636
+msgid "Cannot split {:d} X points equally between {:d} processors\n"
+msgstr ""
+
+#: ../src/bout++.cxx:736
+msgid "Check if a file exists, and exit if it does."
+msgstr ""
+
+#: ../src/bout++.cxx:415
+#, fuzzy
+msgid ""
+"Code compiled on {:s} at {:s}\n"
 "\n"
 msgstr ""
 "代码于 %s %s 编译\n"
 "\n"
 
-#: src/bout++.cxx:256
-#, c-format
-msgid "DataDir \"%s\" does not exist or is not accessible\n"
+#: ../src/sys/optionsreader.cxx:122
+msgid "Command line"
+msgstr ""
+
+#: ../tests/unit/src/test_bout++.cxx:300 ../src/bout++.cxx:425
+msgid "Compile-time options:\n"
+msgstr ""
+
+#: ../tests/unit/src/test_bout++.cxx:304
+msgid "Compiled with flags"
+msgstr ""
+
+#: ../src/mesh/impls/bout/boutmesh.cxx:882
+msgid "Constructing default regions"
+msgstr ""
+
+#: ../src/bout++.cxx:400
+msgid "Could not create PID file {:s}"
+msgstr ""
+
+#: ../src/mesh/impls/bout/boutmesh.cxx:398
+msgid ""
+"Could not find a valid value for NXPE. Try a different number of processors."
+msgstr ""
+
+#: ../src/sys/options/options_ini.cxx:127
+msgid "Could not open output file '{:s}'\n"
+msgstr ""
+
+#: ../src/bout++.cxx:534
+msgid "Could not open {:s}/{:s}.{:d} for writing"
+msgstr ""
+
+#. Error reading
+#: ../src/mesh/mesh.cxx:466
+msgid "Could not read integer array '{:s}'\n"
+msgstr ""
+
+#. Failed . Probably not important enough to stop the simulation
+#: ../src/bout++.cxx:514
+msgid "Could not run bout-log-color. Make sure it is in your PATH\n"
+msgstr ""
+
+#: ../src/solver/solver.cxx:590
+msgid "Couldn't add Monitor: {:g} is not a multiple of {:g}!"
+msgstr ""
+
+#: ../src/mesh/mesh.cxx:500
+msgid "Couldn't find region {:s} in regionMap2D"
+msgstr ""
+
+#: ../src/mesh/mesh.cxx:492
+msgid "Couldn't find region {:s} in regionMap3D"
+msgstr ""
+
+#: ../src/mesh/mesh.cxx:508
+msgid "Couldn't find region {:s} in regionMapPerp"
+msgstr ""
+
+#: ../src/sys/options.cxx:267
+msgid "Couldn't get BoutReal from option {:s} = '{:s}'"
+msgstr ""
+
+#: ../src/sys/options.cxx:216
+msgid "Couldn't get integer from option {:s} = '{:s}'"
+msgstr ""
+
+#: ../src/bout++.cxx:388
+#, fuzzy
+msgid "DataDir \"{:s}\" does not exist or is not accessible\n"
 msgstr "\"%s\" 不存在或不可访问\n"
```

Comment:
Can the .po files be searched-and-replaced to change %s -> {:s} etc. in `msgstr` entries?

Reply:
Done in PR #1866